src/lib/router.js

```diff
@@ -52,7 +61,7 @@ async function swapContent(url) {
 
 router
   .on("/", async () => {
-    return swapContent("index.html");
+    return swapContent("");
```

Comment:
When we dispatch the pageView event in the swapContent function, I think the url for the page will be blank. Seems like we should add a condition there.

Reply:
This just matches the other URLs, like "measure", "learn", which don't have a prefix "/".

What do you want to do? Something special?

Reply:
Ah I think it's a moot point because `ga("send", "pageview")` doesn't actually pass along the url (I guess it's using the location).

I thought the next route, `/*`, is what matches pages like /measure and /learn. In those cases it seems fine since `'measure'` and `'learn'` are passed as the `url`. So when the handler calls:

```
document.dispatchEvent(new CustomEvent("pageview", {detail: url})); 
```

it works as expected.

In the example I highlighted, since `url` is an empty string, I thought we were telling google analytics to track the url `""` instead of `"/"`. My mistake 😅